client/components/deposits-overview/test/index.tsx

```diff
@@ -216,62 +218,6 @@ describe( 'Deposits Overview information', () => {
 		);
 		expect( getByText( 'October 1, 2021' ) ).toBeTruthy();
 	} );
-
-	test( 'Confirm next deposit default status and date', () => {
-		const overview = createMockOverview( 'usd', 100, 0, 'rubbish' );
-
-		const { getByText } = render(
-			<NextDepositDetails isLoading={ false } overview={ overview } />
-		);
-		expect( getByText( 'Unknown' ) ).toBeTruthy();
-		expect( getByText( '—' ) ).toBeTruthy();
-	} );
```

Comment:
This test confirmed that a deposit with the 'rubbish' status would result in a "Unknown" status pill. This isn't possible any more with the strict type.